build.proj

```diff
@@ -39,9 +39,15 @@
       <MSBuild Projects="tools-local/Microsoft.DotNet.Build.Tasks.Local/Microsoft.DotNet.Build.Tasks.Local.builds" />
       <MSBuild Projects="tools-local/tasks/core-setup.tasks.builds" />
 
+    <PropertyGroup>
+      <DependencyModelTFM>netstandard1.3</DependencyModelTFM>
+      <DependencyModelTFM Condition="'$(BuildToolsTargets45)' == 'true' OR '$(MSBuildRuntimeType)' != 'Core'">net451</DependencyModelTFM>
```

Comment:
We have generally been moving away from BuildToolsTargets45 and instead just using the MSBuildRuntimeType check.